front/main_local.html

```diff
@@ -80,7 +80,7 @@
                     <div class="panel panel-default">
                         <div class="panel-body">
                             <div id="canvas-parent">
-                                <div id="img_url" hidden>http://www.inright.ru/media/55_1.jpg</div>
+                                <div id="img_url" hidden>../build/shot.png</div>
```

Comment:
please, specify a file that is either in the repository, or can be got by a link